docs/Editing/Toggling and Editing Statuses.md

```diff
@@ -79,6 +79,8 @@ Obsidian allows you to assign a [hotkey](https://help.obsidian.md/Customization/
 | In Tasks query search results | Live Preview | ❌     |
 | In Tasks query search results | Reading mode | ❌     |
 
+This command can also be triggered programmatically via the [[Tasks Api#`executeToggleTaskDoneCommand (line string, path string) => string;`|Tasks API]].
+
```

Comment:
@claremacrae do you think we need to add a `released` section here? I would say we don't since there is a link to where the API description with the `released` section inside =)

Reply:
Yes, I feel it's worth it... I would say something like,  'Since Tasks X.Y.Z, this command can...'

Reply:
Added!